usr/lib/common/key_mgr.c

```diff
@@ -1358,6 +1358,18 @@ CK_RV key_mgr_unwrap_key(STDLL_TokData_t *tokdata,
         goto done;
     }
 
+    /* Give the token a chance to check if it supports the unwrapped key */
+    if (token_specific.t_set_attrs_for_new_object != NULL) {
+        rc = token_specific.t_set_attrs_for_new_object(tokdata, keyclass,
+                                                       MODE_UNWRAPPED,
+                                                       key_obj->template);
+        if (rc != CKR_OK) {
+            TRACE_ERROR("token_specific.t_set_pkey_attr failed with rc=%lx\n",
```

Comment:
It's "token_specific.t_set_attrs_for_new_object" (not token_specific.t_set_pkey_attr)

Reply:
fixed